src/compiler/_namespaces/ts.ts

```diff
@@ -68,7 +68,5 @@ export * from "../watch";
 export * from "../watchPublic";
 export * from "../tsbuild";
 export * from "../tsbuildPublic";
-import * as moduleSpecifiers from "./ts.moduleSpecifiers";
-export { moduleSpecifiers };
-import * as performance from "./ts.performance";
-export { performance };
+export * as moduleSpecifiers from "./ts.moduleSpecifiers";
+export * as performance from "./ts.performance";

```

Comment:
Pretty much all import linters want to reformat the old code to split imports from exports. I had kept the old form because tooling (api-extractor, when I was using it, some bundlers) seemed to do a better job with it, but I swapped it to the more concice form in this PR so that reformatting imports works better. Maybe this is a bad idea and I should just undo these changes by hand.

Reply:
I prefer the more concise form if none of our tooling requires the more verbose form, too.

Reply:
I agree, the concise form is better.

I did a diff to see, and curiously this does actually seem to cause an output difference in esbuild. When we use this short, it's unable to remove some of the export objects, leaving things like:


```
  // src/jsTyping/_namespaces/ts.server.ts
  var ts_server_exports = {};
  __export(ts_server_exports, {
    ActionInvalidate: () => ActionInvalidate,
    ActionPackageInstalled: () => ActionPackageInstalled,
    ActionSet: () => ActionSet,
    Arguments: () => Arguments,
    EventBeginInstallTypes: () => EventBeginInstallTypes,
    EventEndInstallTypes: () => EventEndInstallTypes,
    EventInitializationFailed: () => EventInitializationFailed,
    EventTypesRegistry: () => EventTypesRegistry,
    findArgument: () => findArgument,
    hasArgument: () => hasArgument,
    nowString: () => nowString
  });

```

Even though it's unreferenced. It's not a correctness problem, but is less efficient.

This made me remember that I did notice this in the process of the module conversion and that's why I left it as the less concise form. I was never able to minimize it.

I will probably revert these as to not regress our output and try and report it upstream.